MQ2PluginManager.cpp

```diff
@@ -94,6 +100,78 @@ void DoPluginTool(PSPAWNINFO pChar, PCHAR szLine)
 	}
 }
 
+static MQPlugin* FindMQ2Plugin(const char* szLine)
+{
+	MQPlugin* pPlugin = pPlugins;
+	while (pPlugin)
+	{
+		if (!_stricmp(const_cast<char*>(szLine), pPlugin->szFilename))
+		{
+			return pPlugin;
+		}
+		pPlugin = pPlugin->pNext;
+	}
+	return nullptr;
+}
+
+
+void DrawPluginManager_MQSettingsPanel() {
```

Comment:
Please note that the rest of the code in this file starts braces on the next line, and you should follow that same pattern in your code here.